joplin/base/models/official_documents_page.py

```diff
@@ -39,6 +39,7 @@ class OfficialDocumentPage(JanisBasePage):
         InlinePanel('topics', label='Topics'),
         InlinePanel('related_departments', label='Related Departments'),
         InlinePanel('official_documents', label="Documents", heading="Entries will be listed by document date (newest first)."),
+        InlinePanel('janis_urls', label='Urls'),
```

Comment:
Whats the thinking behind them being in the author interface?

Reply:
I wanted to see them without going into the console (and be able to show content authors what URLs are being generated)

I'm very open to removing them completely, but I was hoping to chat with @desigonz to figure out if we might want to show them somewhere